sdk/identity/identity/integration/AzureKubernetes/Dockerfile

```diff
@@ -10,11 +10,12 @@ ARG NODE_VERSION=20
 ARG REGISTRY=""
 
 FROM ${REGISTRY}node:${NODE_VERSION}-alpine as repo
-RUN apk --no-cache add git
-RUN git clone https://github.com/azure/azure-sdk-for-js --single-branch --branch main --depth 1 /azure-sdk-for-js
+WORKDIR /app
 
-WORKDIR /azure-sdk-for-js/sdk/identity/identity/test/integration/AzureKubernetes
-RUN npm install
-RUN npm install -g typescript
-RUN tsc -p .
-CMD ["node", "index"]
+COPY . .
```

Comment:
Did you mean to put 2 dots here?

Reply:
Yup! 

The COPY command copies from src to dest

`.` in src is the working directory where Dockerfile exists. `.` in dest is the cwd set by WORKDIR above.

Hope that helps! https://docs.docker.com/reference/dockerfile/#copy